CHANGELOG.md

```diff
@@ -57,6 +57,7 @@
 * [CHANGE] Removed `-store.fullsize-chunks` option which was undocumented and unused (it broke ingester hand-overs). #2656
 * [CHANGE] Query with no metric name that has previously resulted in HTTP status code 500 now returns status code 422 instead. #2571
 * [CHANGE] Experimental Delete Series: Change target flag for purger from `data-purger` to `purger`. #2777
+* [CHANGE] Ingester: Chunks flushed via /flush stay in memory until retention period is reached. This affects `cortex_ingester_memory_chunks` metric. #2778
```

Comment:
I can do that, but there are already changes merged on master, that are not in 1.2-rc0, so cleanup needs to happen when merging release branch to master. (Personally I think we should do the first merge as soon as 1.2-rc0 is tagged, to avoid this problem growing. It is a pain to clean up.)